src/components/CampaignsList/CampaignsList.js

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
+function CampaignList(props) {
+  return (
+    <div>
+      <h1>Available Campaigns</h1>
+      <ul>
+        {props.campaigns.map((campaign) => (
+          <li key={campaign._id}>
```

Comment:
everything looks good, but should we do a <Link key={campaign._id} to={`/campaigns/${campaign._id}`}> to make it a clickable link to its details page